modules/simulation/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
+ev_add_module(DCSupplySimulator)
```

Comment:
We could remove JsDCSupplySimulator probably with this PR and update the configs in core to use the C++ version

Reply:
Not yet, as the car sides of the SIL configs use JsDCSupplySimulator's `powermeter` interface implementation. That is not covered in this PR. (Should it be?)

Reply:
Ah right, not neccesarily now, but I'd like to drop the JS module at some point